lighthouse-core/report/v2/renderer/report-renderer.js

```diff
@@ -147,13 +180,58 @@ class ReportRenderer {
 
   /**
    * @param {!ReportJSON} report
+   * @return {!DocumentFragment}
+   */
+  _renderReportHeader(report) {
+    const header = this._dom.cloneTemplate('#tmpl-lh-heading');
+    header.querySelector('.lh-config__timestamp').textContent =
+        formatDateTime(report.generatedTime);
+    const url = header.querySelector('.lh-metadata__url');
+    url.href = report.url;
+    url.textContent = report.url;
+
+    const env = header.querySelector('.lh-env__items');
+    report.runtimeConfig.environment.forEach(runtime => {
+      const item = this._dom.cloneTemplate('#tmpl-lh-env__items', env);
+      item.querySelector('.lh-env__name').textContent = runtime.name;
+      item.querySelector('.lh-env__description').textContent = runtime.description;
+      item.querySelector('.lh-env__enabled').textContent =
+          runtime.enabled ? 'Enabled' : 'Disabled';
+      env.appendChild(item);
+    });
+
+    return header;
+  }
+
+  /**
+   * @param {!ReportJSON} report
+   * @return {!DocumentFragment}
+   */
+  _renderReportFooter(report) {
+    const footer = this._dom.cloneTemplate('#tmpl-lh-footer');
+    footer.querySelector('.lh-footer__version').textContent = report.lighthouseVersion;
+    footer.querySelector('.lh-footer__timestamp').textContent =
+        formatDateTime(report.generatedTime);
+    return footer;
+  }
+
+  /**
+   * @param {!ReportJSON} report
    * @return {!Element}
    */
   _renderReport(report) {
-    const element = this._dom.createElement('div', 'lh-report');
+    const container = this._dom.createElement('div', 'lh-content');
```

Comment:
there's a `.lh-content` div inside a `#lh-content` div? Or is this a old code? Appears to be discarded since `element` is returned and attached elsewhere in the DOM